gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -476,6 +476,9 @@ private void cancelDag(String dagToCancel) throws ExecutionException, Interrupte
         for (DagNode<JobExecutionPlan> dagNodeToCancel : dagNodesToCancel) {
           cancelDagNode(dagNodeToCancel);
         }
+
+        this.dags.get(dagToCancel).setFlowEvent(TimingEvent.FlowTimings.FLOW_CANCELLED);
```

Comment:
Should we setFlowEvent() inside cancelDagNode() ?

Reply:
This. is setting the flow event on a dag level, not a dag node level, so it makes more sense to do it in cancelDag.